rs/consensus/src/idkg/signer.rs

```diff
@@ -1548,6 +1551,7 @@ mod tests {
                 fake_schnorr_master_public_key_id(SchnorrAlgorithm::Ed25519)
             }
             MasterPublicKeyId::Schnorr(_) => fake_ecdsa_master_public_key_id(),
+            MasterPublicKeyId::VetKd(_) => fake_vetkd_master_public_key_id(),
```

Comment:
I think for consistency we should also panic here
```suggestion
            MasterPublicKeyId::VetKd(_) => panic!("not applicable to vetKD"),
```

Reply:
Done.